src/rabbit_exchange_type_lvc.erl

```diff
@@ -26,37 +24,18 @@ description() ->
 
 serialise_events() -> false.
 
-route(#exchange{name = Name}, Msg, _Opts) ->
+route(#exchange{name = Name}, Msg, Opts) ->
```

Comment:
Opts is unused, doesn't the compiler complain?

Reply:
Mmmm, weird, for some reason it didn't. It had complained about some other errors, but not about that one.